app/models/queue_item.rb

```diff
@@ -20,4 +29,9 @@ def category_name
   def category
     video.category
   end
+
+  private
+  def review
+    @review ||= Review.where(user_id: user.id, video_id: video.id).first
```

Comment:
We strongly recommend `find_by` over `where(...).first`

HOMEWORK: Check lesson 1 assignments and search for notes, where it is mentioned to use `find_by`.